src/test/app.test.js

```diff
@@ -4,9 +4,9 @@ import _ from 'lodash';
 import fs from 'fs-extra';
 import path from 'path';
 import {assert, test as helpers} from 'yeoman-generator';
-
 import * as libs from './libs';
 import packageJSON from '../generators/app/package-json';
+import {assert as chaiAssert} from 'chai';
```

Comment:
I just looked at the yeoman api docs and didn't see anything, but I never realized that it inherits from node assert. I will look into it